Frameworks/Account/Account.swift

```diff
@@ -298,7 +298,9 @@ public final class Account: DisplayNameProvider, UnreadCountProvider, Container,
 	}
 
 	public func refreshAll(completion: @escaping (Result<Void, Error>) -> Void) {
-		self.delegate.refreshAll(for: self, completion: completion)
+		delegate.sendArticleStatus(for: self) { [unowned self] in
+			self.delegate.refreshAll(for: self, completion: completion)
+		}
 	}
```

Comment:
@tgt Could you please move the sendArticleStatus into the FeedbinAccountDelegate in the refreshAll func?  Right between the refreshArticles and the refreshArticleStatus is where it should have been put.  This will keep the meaning of refreshAll for the Account to mean the same thing as the refreshAll for the AccountDelegate.